packages/chain-space/src/ChainSpace.chain.ts

```diff
@@ -308,6 +309,42 @@ export async function dispatchSubspaceCreateToChain(
     authorization: chainSpace.authorizationUri,
   }
 
+  try {
+    
+    const extrinsic = await prepareCreateSubSpaceExtrinsic(
+      chainSpace,
+      authorAccount,
+      count,
+      parent,
+      creatorUri,
+      signCallback
+    )
+    await Chain.signAndSubmitTx(extrinsic, authorAccount)
+
+    return returnObject;
+  } catch (error) {
+    throw new SDKErrors.CordDispatchError(
+      `Error dispatching to chain: "${error}".`
+    )
+  }
+}
+
+/**
+ * Prepares the creation of a sub-space extrinsic for later dispatch to the blockchain.
+ * @param chainSpace - The ChainSpace object containing necessary information for creating the ChainSpace on the blockchain.
+ * @param authorAccount - The blockchain account used for signing and submitting the transaction.
+ * @param count - The count of subspaces to be created.
+ * @param creatorUri - The DID URI of the creator, used to authorize the transaction.
+ * @returns The prepared extrinsic ready for batch signing and submitting.
+ */
+export async function prepareCreateSubSpaceExtrinsic(
+  chainSpace: IChainSpace,
+  authorAccount: CordKeyringPair,
+  count: number,
```

Comment:
`count` here refers to count of transactions permitted to be performed on the chain.

Reply:
Updated, thanks for the review!